Documentation/ceph-filesystem-crd.md

```diff
@@ -94,6 +96,16 @@ spec:
     activeStandby: true
 ```
 
+**IMPORTANT**: For erasure coded pools, we have to create a replicated pool as the primary pool and erasureCoded as the secondary pool.
+Separately, a directory layout property needs to be added to use the secondary EC data pool.
+
+```console
```

Comment:
They would first need to connect to the application pod before doing this, right?